package.json

```diff
@@ -85,20 +85,14 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.15.1",
     "@popperjs/core": "^2.9.3",
+    "@types/d3-hexbin": "^0.2.3",
+    "@types/d3-selection": "^1.3.0",
     "bootstrap": "~5.1.0",
     "crypto-js": "^4.0.0",
-    "d3": "~3.5.17",
-    "d3-array": "^2.7.1",
-    "d3-axis": "^1.0.8",
-    "d3-brush": "^1.0.4",
-    "d3-drag": "^1.2.1",
-    "d3-format": "^1.3.0",
-    "d3-polygon": "^1.0.3",
-    "d3-quadtree": "^1.0.3",
-    "d3-scale": "^2.1.0",
-    "d3-selection": "^1.3.0",
-    "d3-shape": "^1.2.0",
-    "d3-zoom": "^1.7.1",
+    "d3-hexbin": "^0.2.2",
+    "d3v3": "npm:d3@~3.5.17",
+    "d3v4": "npm:d3@~4.13.0",
+    "d3v7": "npm:d3@~7.3.0",
```

Comment:
@puehringer Do you agree with the new D3 imports/aliases? Zach merged the former separate d3v4 imports into a single `d3v4` import. Are we then still fine with the bundle size?

Reply:
I think the bundle size is a neglible caveat of this solution, but we should still move forward, as this would ideally give us a way of removing d3[v3|v4] in a more structured fashion. Also, the d3v4 dependency will not be there for long, as it is coming from tdp_scatterplot, something that is deprecated and will be removed soonish.

Reply:
Ok, sounds good.